lms/static/js/student_account/enrollment.js

```diff
@@ -9,7 +9,7 @@ var edx = edx || {};
     edx.student.account.EnrollmentInterface = {
 
         urls: {
-            enrollment: '/api/enrollment/v1/enrollment',
+            enrollment: '/commerce/orders/',
```

Comment:
I'd change this key name from `enrollment` to `orders` to more accurately reflect the behavior of the URL it now corresponds to.